ohsome/clients.py

```diff
@@ -29,11 +32,12 @@
 class _OhsomeBaseClient:
     def __init__(
         self,
-        base_api_url=None,
-        log=DEFAULT_LOG,
-        log_dir=DEFAULT_LOG_DIR,
-        cache=None,
-        user_agent=None,
+        base_api_url: str = None,
+        log: bool = DEFAULT_LOG,
+        log_dir: Union[str, Path] = DEFAULT_LOG_DIR,
+        cache: list = None,
+        user_agent: str = None,
+        retry: Retry = None,
```

Comment:
Consider using the `Optional` type to denote that the Retry field is not required.